src/util/redisConnector.js

```diff
@@ -72,20 +73,33 @@ const RedisDB = {
   /**
    * Used to get value from redis depending on the key and the expected value type
    * @param {*} key key for which value needs to be extracted
-   * @param {*} isJsonExpected false if fetched value can not be json
+   * @param {*} isObjExpected false if fetched value can not be json
    * @returns value which can be json or string or number
    *
    */
-  async getVal(key, isJsonExpected = true) {
+  async getVal(key, isObjExpected = true) {
     try {
       await this.checkAndConnectConnection(); // check if redis is connected and if not, connect
-      const value = await this.client.get(key);
-      if (value) {
-        const bytes = Buffer.byteLength(value, 'utf-8');
-        stats.gauge('redis_get_val_size', bytes, {
-        });
+      let value;
+      if (isObjExpected === true) {
+        value = await this.client.hmget(key);
+        if (isDefinedAndNotNull(value)) {
+          Object.keys(value).forEach(Key => {
```

Comment:
Key -> keep key (lower case)

Reply:
we already have `key` as parameter . will change this to `objKey`

Reply:
done